tls/extensions/s2n_client_key_share.c

```diff
@@ -76,7 +76,7 @@ int s2n_extensions_client_key_share_recv(struct s2n_connection *conn, struct s2n
     uint16_t named_group, share_size;
     int supported_curve_index;
 
-    int bytes_processed = 0;
+    uint32_t bytes_processed = 0;
```

Comment:
Comment explaining why?

Reply:
Just pushed a commit to add said comment